### PR TITLE
Feature/day 4

### PR DESCRIPTION
**Pull request by tejaschapke18**

### CODE REVIEW

**Review by Sandip-rana-successive — CHANGES_REQUESTED** [review]
Please make the changes mentioned in the comment

File: Node-Backend/src/Middleware/authMiddleware.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
+const jwt = require("jsonwebtoken");
+
+const secretKey =
+  "b44fd2de00412db5ebc7350536b59e86731142f100deef1d486972b9c22e6b11";
```
[the remark this answers is not in the document]
Comment (tejaschapke18) [reply Node-Backend/src/Middleware/authMiddleware.js]: Added secret key in env

File: Node-Backend/src/Middleware/errorHandler.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+const errorHandler = (err, req, res, next) => {
+  let statusCode = err.statusCode || 500;
```
[the remark this answers is not in the document]
Comment (tejaschapke18) [reply Node-Backend/src/Middleware/errorHandler.js]: Added optional chaining

File: Node-Backend/src/Middleware/errorHandler.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+  let errorMessage = err.message || "Internal Server Error";
+  if (!err.statusCode && !err.message) {
```
[the remark this answers is not in the document]
Comment (tejaschapke18) [reply Node-Backend/src/Middleware/errorHandler.js]: removed!

**Review by Sandip-rana-successive — APPROVED** [review]
LGTM

**Pull request merged**